genesyscloud/flow_loglevel/resource_genesyscloud_flow_loglevel_test.go

```diff
@@ -0,0 +1,112 @@
+package flow_loglevel
+
+import (
+	"fmt"
+	"github.com/google/uuid"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+	"github.com/mypurecloud/platform-client-sdk-go/v125/platformclientv2"
+	"terraform-provider-genesyscloud/genesyscloud/architect_flow"
+	"terraform-provider-genesyscloud/genesyscloud/provider"
+	"terraform-provider-genesyscloud/genesyscloud/util"
+	"testing"
+)
+
+func TestAccResourceFlowLogLevel(t *testing.T) {
+	var (
+		flowResource         = "test_logLevel_flow1"
+		resourceId           = "flow_log_level" + uuid.NewString()
+		flowName             = "Terraform Test Flow log level " + uuid.NewString()
+		flowLoglevelBase     = "Base"
+		flowLoglevelAll      = "All"
+		flowLogLevelDisabled = "Disabled"
+		flowId               = "${genesyscloud_flow." + flowResource + ".id}"
+		filePath             = "../../examples/resources/genesyscloud_flow/inboundcall_flow_example.yaml"
+		inboundCallConfig    = fmt.Sprintf("inboundCall:\n  name: %s\n  defaultLanguage: en-us\n  startUpRef: ./menus/menu[mainMenu]\n  initialGreeting:\n    tts: Archy says hi!!!\n  menus:\n    - menu:\n        name: Main Menu\n        audio:\n          tts: You are at the Main Menu, press 9 to disconnect.\n        refId: mainMenu\n        choices:\n          - menuDisconnect:\n              name: Disconnect\n              dtmf: digit_9", flowName)
+	)
+
+	flowResourceConfig := architect_flow.GenerateFlowResource(
+		flowResource,
+		filePath,
+		inboundCallConfig,
+		true,
+	)
+
+	resource.Test(t, resource.TestCase{
+		PreCheck: func() {
+			util.TestAccPreCheck(t)
+		},
+		ProviderFactories: provider.GetProviderFactories(providerResources, providerDataSources),
+		Steps: []resource.TestStep{
+			{
+				// Create using flow log level Base
+				Config: flowResourceConfig + generateFlowLogLevelResource(
+					flowId,
+					flowLoglevelBase,
+					resourceId,
+				),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr("genesyscloud_flow_loglevel."+resourceId, "flow_log_level", flowLoglevelBase),
+				),
+			},
+			{
+				// Create using flow log level All
```

Comment:
This test step will actually update the resource. Resources are not deleted between test steps so for each test step after the first one, the update function is called. You don't need to change the test or anything I would just update the comment to say update, same for the one below. I know this is just a comment but I don't want someone to look at this is the future and be confused.